src/hooks/usePipelineRuns.ts

```diff
@@ -119,12 +119,20 @@ const useRuns = <Kind extends K8sResourceCommon>(
     NextPageProps,
   ];
 
-  return React.useMemo(() => {
-    const rResources =
+  const rResources = React.useMemo(() => {
+    const combined =
       runs && trResources
         ? uniqBy([...runs, ...trResources], (r) => r.metadata.name)
         : runs || trResources;
 
+    return combined?.sort((a, b) => {
+      const aTime = (a.metadata.creationTimestamp || a.status?.startTime || '') as string;
+      const bTime = (b.metadata.creationTimestamp || b.status?.startTime || '') as string;
+      return bTime.localeCompare(aTime);
+    });
+  }, [runs, trResources]);
```

Comment:
Sort the data out of usePipelineRuns hook, maybe in PipelineRunListView component

Reply:
I updated back to what I implemented initially (sort in the PipelineRunListView component)